SentinelOne XSOAR Mirroring (#24319)
demisto/content: Packs/SentinelOne/ReleaseNotes/3_1_0.md

#### Integrations
##### SentinelOne v2
- Updated the Docker image to: *demisto/python3:3.10.10.48392*.

> - Added support for Mirroring Integration. This integration allows us to mirror incidents/tickets/cases from SentinelOne. This feature applicable to the users who had the Cortex XSOAR version 6.0.0 and more. In order to achieve Mirroring added following things:

Reviewer: ```suggestion
- Added support for the Mirroring integration. This integration allows us to mirror incidents/tickets/cases from SentinelOne. This feature is applicable for users who have Cortex XSOAR version 6.0.0 and more. In order to achieve mirroring added following things:
```

    - Added a new field `Incident Mirroring Direction` to config.
    - Added a `Mirror SentinelOne Ticket` playbook which enables ticket mirroring with SentinelOne.

>     - Added a `SentinelOne Incident Layout` which is a custom layout only for the Mirroring.

Reviewer: ```suggestion
    - Added a `SentinelOne Incident Layout` which is a custom layout only for mirroring.
```

    - Added a `SentinelOne Incident` type.

>     - Added an incoming and outgoing mappers.

Reviewer: ```suggestion
    - Added an incoming and outgoing mapper.
```

    - Added a few custom Incident Fields.

> - **SentinelOne Threat Analyst Verdict**

Reviewer: ```suggestion
   - **SentinelOne Threat Analyst Verdict**
```

> - **SentinelOne Threat ID**

Reviewer: ```suggestion
   - **SentinelOne Threat ID**
```

- **SentinelOne Threat Status**.

#### Mappers
##### New: SentinelOne - Outgoing Mapper
- SentinelONe Outgoing Mapper for the Sentinelone Incidents.

##### New: SentinelOne Incoming Mapper
- SentinelONe Incoming Mapper for the Sentinelone Incidents.

#### Classifiers
##### New: SentinelOne Classifier
- Classifies SentinelOne incidents.

#### Layouts
##### New: SentinelOne Incident Layout

> - A custom Layout for the SentinelOne Incident.

Reviewer: ```suggestion
A custom Layout for the SentinelOne incident.
```
